bin/s3-upload

```diff
@@ -113,7 +116,7 @@ for (( i=0; i< S3_ENTRIES; i++ )); do
     LOCAL_PATH="$PARENT_PATH"/$(echo "$JSON_CONTEXT" | jq -r '.content.s3['$i'].path')
     S3_BUCKET=$(echo "$JSON_CONTEXT" | jq -r '.content.s3['$i'].bucket')
     S3_KEY=$(echo "$JSON_CONTEXT" | jq -r '.content.s3['$i'].key')
-    echo "NOTICE Processing file at $LOCAL_PATH (s3_bucket=$S3_BUCKET s3_key=$S3_KEY)"
+    echo "INFO Processing file at $LOCAL_PATH (s3_bucket=$S3_BUCKET s3_key=$S3_KEY)"
```

Comment:
it would be better to have this as debug, and have an info message based on the success of the operation